static/app/views/replays/detail/layout/index.tsx

```diff
@@ -108,6 +121,23 @@ function ReplayLayout({
     );
   }
 
+  if (layout === 'sidebar_left') {
+    return (
+      <Container>
+        {timeline}
+        <PageRow>
+          <ResizePanel direction="e" minWidth={SIDEBAR_MIN_WIDTH}>
+            <SidebarSection>
+              {video ? <ResizePanel direction="s">{video}</ResizePanel> : null}
+              {crumbs}
+            </SidebarSection>
```

Comment:
This is going to need a rebase & update since https://github.com/getsentry/sentry/pull/36302 got merged first.

```diff
              <SidebarSection>
+               <AsideTabsV2 showCrumbs={showCrumbs} showVideo={showVideo} />
-               {video ? <ResizePanel direction="s">{video}</ResizePanel> : null}
-               {crumbs}
            </SidebarSection>
```

Reply:
bump